docs/pages/workflow/android-studio-emulator.mdx

```diff
@@ -3,55 +3,82 @@ title: Android Studio Emulator
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
 
-If you don't have an Android device available to test with, we recommend using the default emulator that comes with Android Studio. If you run into any problems setting it up, follow the steps in this guide.
+If you don't have an Android device available to test with, we recommend using the default emulator that comes with Android Studio.
+If you run into any problems setting it up, follow the steps in this guide.
 
-## Step 1: Set up Android Studio's tools
+## Set up Android Studio's tools
 
-- [Download](https://developer.android.com/studio) and install Android Studio 3.0+.
+<Step label="1">
 
-- Select "Standard" for the "Install Type" inside the wizard.
+[Download](https://developer.android.com/studio) and install Android Studio 3.0+.
 
-- Inside Android Studio, go to Preferences > Appearance & Behavior > System Settings > Android SDK. Click on the "SDK Tools" tab and make sure you have at least one version of the "Android SDK Build-Tools" installed.
+</Step>
+
+<Step label="2">
+
+Select **"Standard"** for the **"Install Type"** inside the wizard.
+
+</Step>
+
+<Step label="3">
+Inside Android Studio, go to **Preferences > Appearance & Behavior > System Settings > Android SDK**.
+Click on the **"SDK Tools"** tab and make sure you have at least one version of the **"Android SDK Build-Tools"** installed.
 
 <ImageSpotlight
   alt="Android SDK build tools"
   src="/static/images/android-studio-build-tools.png"
-  containerStyle={{ paddingBottom: 0 }}
 />
+</Step>
 
-- Copy or remember the path listed in the box that says "Android SDK Location."
+<Step label="4">
+Copy or remember the path listed in the box that says **"Android SDK Location"**.
 
 <ImageSpotlight
   alt="Android SDK location"
   src="/static/images/android-studio-sdk-location.png"
-  containerStyle={{ paddingBottom: 0 }}
 />
+</Step>
 
-- If you are on macOS or Linux, add an [environment variable](https://developer.android.com/studio/command-line/variables#envar) pointing to the Android SDK location in `~/.bash_profile` (or `~/.zshenv` if you use Zsh) - eg. `export ANDROID_HOME=/your/path/here`. Copy and paste these two lines to do this automatically for Bash and Zsh:
+<Step label="5">
+If you are on macOS or Linux, add an [environment variable](https://developer.android.com/studio/command-line/variables#envar) pointing to the Android SDK location
+in `~/.bash_profile` (or `~/.zshenv` if you use Zsh) - eg. `export ANDROID_HOME=/your/path/here`. Copy and paste these two lines to do this automatically for Bash and Zsh:
```

Comment:
```suggestion
in `~/.bash_profile` (or `~/.zshenv` if you use Zsh) - for example:`export ANDROID_HOME=/your/path/here`. Copy and paste these two lines to do this automatically for Bash and Zsh:
```

Unrelated to the changes in this PR.
I'm suggesting here to update Latin abbreviations to spell out the word it is meant for as per our [docs styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations).